bot/command_error_handlers/command_not_found.py

```diff
@@ -0,0 +1,64 @@
+from typing import NoReturn
+
+from discord import Embed, Interaction, errors
+from discord.ext import commands
+from pydis_core.utils.error_handling.commands import AbstractCommandErrorHandler
+from pydis_core.utils.logging import get_logger
+
+from bot.bot import Bot
+from bot.utils.commands import get_command_suggestions
+
+log = get_logger(__name__)
+
+DELETE_DELAY = 10
+QUESTION_MARK_ICON = "https://cdn.discordapp.com/emojis/512367613339369475.png"
+
+
+class CommandNotFoundErrorHandler(AbstractCommandErrorHandler):
+    """A handler for all CommandNotFound exceptions."""
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    async def should_handle_error(self, error: errors.DiscordException) -> bool:
```

Comment:
Thoughts about using `@typing.override` for these? It makes it more explicit that this is an override, and it helps type checkers. But if we don't use a type checker then it could be argued that this would just clutter the code.